src/wireviz/wireviz.py

```diff
@@ -232,16 +245,21 @@ def main():
         print(f'Error: input file {args.input_file} inaccessible or does not exist, check path')
         sys.exit(1)
 
-    with open_file_read(args.input_file) as fh:
-        yaml_input = fh.read()
-
-    if args.prepend_file:
-        if not os.path.exists(args.prepend_file):
-            print(f'Error: prepend input file {args.prepend_file} inaccessible or does not exist, check path')
-            sys.exit(1)
-        with open_file_read(args.prepend_file) as fh:
-            prepend = fh.read()
-            yaml_input = prepend + yaml_input
+    if ".png" in args.input_file:
+        yaml_input = read_yaml_from_png(args.input_file.replace('.png',''))
+        with open(args.input_file.replace('.png','_out.yaml'),'w') as fh:
+            fh.write(yaml_input) # Extract yaml to separate file
+    else:
+        with open_file_read(args.input_file) as fh:
+            yaml_input = fh.read()
+
+        if args.prepend_file:
+            if not os.path.exists(args.prepend_file):
```

Comment:
Consider using `Pathlib`'s `exists()` function instead (see #190)

Reply:
Github is reporting this as a change, but this is mostly whitespace changes only, I only indented here.

Reply:
true!